Orange/widgets/data/tests/test_owsave.py

```diff
@@ -26,7 +26,7 @@ def setUp(self):
     def test_ordinary_save(self):
         self.send_signal(self.widget.Inputs.data, Table("iris"))
 
-        for ext, writer in [('.tab', TabReader), ('.pickle', PickleReader)]:
+        for ext, writer in [('.tab', TabReader), ('.pickle', PickleReader), ('.xls', ExcelReader), ('.xlsx', ExcelReader), ('.ods', ExcelReader)]:
```

Comment:
This line is too long. Specify each tuple (including the first two) on its own line.